### DISCUSSION TITLE
2022/03/13/%E3%80%8E%E6%9D%82%E6%96%87%E3%80%8F%E5%9C%A8%E8%BF%B7%E8%8C%AB%E4%B8%AD%E5%89%8D%E8%BF%9B/

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#16] [General]
# 2022/03/13/%E3%80%8E%E6%9D%82%E6%96%87%E3%80%8F%E5%9C%A8%E8%BF%B7%E8%8C%AB%E4%B8%AD%E5%89%8D%E8%BF%9B/

现在时间是晚上22点36分，每日一篇的技术文肯定是写不完了，就写一篇近期的个人感想吧  很烦，很累 这几天我感觉很累，真的 从周五早上开始放假，再到现在周日晚上，我想了想，我基本就没做什么正事 玩也没有好好玩，学习也没有好好学习，属于一种不在状态的样子，做什么事情都没有劲 为什么呢？我不知道，我

https://nx-official.github.io/2022/03/13/%E3%80%8E%E6%9D%82%E6%96%87%E3%80%8F%E5%9C%A8%E8%BF%B7%E8%8C%AB%E4%B8%AD%E5%89%8D%E8%BF%9B/

**Comment by ZhangStudyLife** [comment]
我对自动化的前途也表示非常的迷茫，不知道自己的兴趣何在，以及害怕进入大学的不适应以及未来的水课该如何应对
